pages/k8s/cni-canal.md

```diff
@@ -0,0 +1,72 @@
+---
+wrapper_template: "base_docs.html"
+markdown_includes:
+  nav: "shared/_side-navigation.md"
+context:
+  title: "CNI with canal"
+  description: How to manage and deploy Kubernetes with Canal
+keywords: CNI, networking
+tags: [operating]
+sidebar: k8smain-sidebar
+permalink: cni-with-canal.html
+layout: [base, ubuntu-com]
+toc: False
+---
+
+'Canal' is a shorthand for saying "Calico and Flannel", a common practise which sets up Calico to handle policy management and Flannel to manage the network itself. The manual setup is explained in the [Calico documentation][canal].  
+
+
+## Deploying Charmed Kubernetes with Canal
+
+To deploy CDK with Canal, deploy the kubernetes-canal bundle:
+
+```bash
+juju deploy cs:bundle/canonical-kubernetes-canal
```

Comment:
is there a newer bundle than this?

Reply:
Nope. This is still the correct and only bundle we publish for CDK with canal.